tests/integration/security/reachability_test.go

```diff
@@ -37,24 +38,22 @@ func TestReachability(t *testing.T) {
 	framework.NewTest(t).
 		Features("security.reachability").
 		Run(func(ctx framework.TestContext) {
-			rctx := reachability.CreateContext(ctx, true)
 			systemNM := istio.ClaimSystemNamespaceOrFail(ctx, ctx)
 
 			testCases := []reachability.TestCase{
 				{
 					ConfigFile: "beta-mtls-on.yaml",
 					Namespace:  systemNM,
 					Include: func(src echo.Instance, opts echo.CallOptions) bool {
-						return true
+						return !apps.Multiversion.Contains(opts.Target)
```

Comment:
Hi John, thanks for reviewing! I have forked this PR and tested without this excluding. You can see the result at https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/28001/integ-security-multicluster-tests_istio/75.

Reply:
Hi John, I have changed this skipping logic into multicluster specific so that we will not reduce the coverage of current `integ-security-k8s-tests_istio`.